{"name": "Негативная проверка удаления курьера с несуществующим id", "status": "passed", "description": "Проверяем, что отправка запроса с несуществующим id курьера возвращает ошибку с кодом ответа - 404и текстом ответа \"Курьера с таким id нет.\"", "steps": [{"name": "Удаляем курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "154895455"}], "start": 1705859708647, "stop": 1705859708766}], "start": 1705859708647, "stop": 1705859708767, "uuid": "922e69a6-f4b0-4ac5-bbff-0e58e06b8bc6", "historyId": "d274b12337d7cefa0fbe96b4af9f699c", "testCaseId": "d274b12337d7cefa0fbe96b4af9f699c", "fullName": "tests.test_delete_courier#test_negative_delete_courier_unexisting_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_delete_courier"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "36652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_delete_courier"}]}